catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/gmti/NitfGmtiTransformer.java

```diff
@@ -66,39 +61,12 @@ public Metacard transform(NitfSegmentsFlow nitfSegmentsFlow, Metacard metacard)
             throw new IllegalArgumentException("argument 'metacard' may not be null.");
         }
 
-        nitfSegmentsFlow.fileHeader(header -> handleHeader(header, metacard))
-                .end();
-
         transformTargetLocation(metacard);
         transformAircraftLocation(metacard);
 
         return metacard;
     }
 
-    private void handleHeader(TaggedRecordExtensionHandler header, Metacard metacard) {
```

Comment:
If you look in the blueprint within imaging-transformer-nitf, around line: 108-111, you can see that we are already extracting nitf header metadata (the NitfHeaderTransformer is called) out of the NITF before the NitfGmtiTransformer is even called. So the code I pulled out was just duplicated and never actually never needed. The NitfHeaderTransformer will also handle mapping any TRE attributes that are in the NITF header.

This class